Pod/Classes/ValueStepper.swift

```diff
@@ -124,6 +118,7 @@ private enum Button: Int {
     
     public required init?(coder aDecoder: NSCoder) {
         super.init(coder: aDecoder)
+        translatesAutoresizingMaskIntoConstraints = false
```

Comment:
IB won't pick up the `intrinsicContentSize`. You need to set it explicitly in the storyboard.